docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -305,6 +309,10 @@
       font-size: hb-calculate-rems(14px);
       padding: hb-calculate-rems(10px);
       width: 100%;
+
+      .ht-pairing-hummingbird & {
+        @include hb-pairing-color('background-color', 'tertiary-reversed');
+      }
```

Comment:
In regards to the color contrast issue on the date / calendar tile, I adjusted this component by targeting the `ht-pairing-hummingbird` class. This isn't something we have done in other color pairing implementations. You can remove these targeted styles so that the `primary` color applies. That will resolve the color contrast issue. Or you could update the color to `tertiary` to match the change made on line 293.

Reply:
If you choose to darken `tertiary-reversed` be sure any applications on its use in the dark footer still pass color contrast as well. :smile: